src/TLuaInterpreter.cpp

```diff
@@ -2190,19 +2190,21 @@ int TLuaInterpreter::disableTimer( lua_State *L )
 
 int TLuaInterpreter::enableKey( lua_State *L )
 {
-    string luaSendText="";
+    string luaNameID = "";
     if( ! lua_isstring( L, 1 ) )
     {
-        lua_pushstring( L, "enableKey: wrong argument type" );
+        lua_pushstring( L, tr( "enableKey: bad argument #1 type (key name as string expected, got %1!)" )
```

Comment:
This _may_ be one source of the current re-basing conflicts and we are no longer planning to translate error messages so these THREE new lines should instead be:

    lua_pushfstring(L, "enableKey: bad argument #1 type (key name as string expected, got %s!)",
                    luaL_typename(L, 1));

and the following pair of lines are also refactored to avoid a moan from [Coverity](https://scan.coverity.com/projects?utf8=%E2%9C%93&search=Mudlet) (or [Codacy](https://www.codacy.com), can't remember which!) about an unreachable line (the `return 1;` one - which is never executed as `lua_error(L)` never returns - as it does a system `longjmp()` instead):

    return lua_error(L);